melodies_monet/util/env_sub.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+def env_sub(file_str):
+    """Substitute envirnoment variables prefixed by $
+    in file location string.
+
+    Parameters
+    -------
+    -------
+    file_str (str): file string
+
+    Returns
+    -------
+    file_str (str): modified file string
+    """
+    for subpath in file_str.split("/"):
+        if "$" in subpath:
+            envvar = subpath.replace("$", "")
+            envval = os.getenv(envvar)
+            if envval is None:
+                print("environment variable not defined: " + subpath)
+                exit(1)
```

Comment:
I think this should raise an Exception instead.
```suggestion
                raise Exception(f"Environment variable {envvar!r} detected in {file_str!r} not defined.")
```